src/main/java/io/getunleash/FakeUnleash.java

```diff
@@ -176,15 +165,5 @@ public List<EvaluatedToggle> evaluateAllToggles(@Nullable UnleashContext context
                                             getVariant(toggleName)))
                     .collect(Collectors.toList());
         }
-
-        @Override
-        public void count(String toggleName, boolean enabled) {
```

Comment:
Public interface change. I don't see a reason why anyone would be using this but it is public. The Yggdrasil engine is now the sink for this data